.github/auto-merge.yml

```diff
@@ -0,0 +1,119 @@
+- match:
+    dependency_name: FakeItEasy
+    update_type: all
+
+- match:
+    dependency_name: FluentValidation
+    update_type: all
+
+- match:
+    dependency_name: Fody
+    update_type: all
+
+- match:
+    dependency_name: GitVersionTask
+    update_type: all
+
+- match:
+    dependency_name: Janitor.Fody
+    update_type: all
+
+- match:
+    dependency_name: McMaster.Extensions.CommandLineUtils
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.Server.Kestrel
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.AspNetCore.StaticFiles
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.Build.Utilities.Core
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.DotNet.Analyzers.Compatibility
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.NET.Test.Sdk
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.PowerShell.5.ReferenceAssemblies
+    update_type: all
+
+- match:
+    dependency_name: Microsoft.SourceLink.GitHub
+    update_type: all
+
+- match:
+    dependency_name: Mono.Cecil
+    update_type: all
+
+- match:
+    dependency_name: Moq
+    update_type: all
+
+- match:
+    dependency_name: NETStandard.Library
```

Comment:
This one is an edge case, and I would probably remove it. Having an update to this dep would be quite a weird thing, and I think if it ever happened, we want to verify why, and if we want it.